spec/data/terraform-config.yaml

```diff
@@ -13,3 +13,13 @@
 
 --- !ruby/object:Provider::Terraform::Config
 overrides: !ruby/object:Overrides::ResourceOverrides
+  ResourceWithTerraformOverride: !ruby/object:Overrides::Terraform::ResourceOverride
```

Comment:
What's the difference between this file and good-tf-override.yaml?

Reply:
good-tf-override.yaml is used to determine whether the overriding is working? for [provider_terraform_resource_override_spec](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/spec/provider_terraform_resource_override_spec.rb) - I'm not against seeing if we can combine them or at least renaming these files (and cleaning any unused files up) so they are less confusing. Would probably do in a different PR